.circleci/config.yml

```diff
@@ -743,7 +743,7 @@ jobs:
 
   deploy_master:
     # build the master branch releasing development docs and wheels
-    <<: *machine_executor
+    executor: ddtrace_dev
```

Comment:
will this still work? we use machine when we need to run docker commands, do we need to here?

Reply:
ah crap yes it does build the wheels in this job. I was thinking maybe we copied them from the other jobs 😕. Ugh.